lib/ansible/modules/cloud/amazon/lambda.py

```diff
@@ -102,6 +102,7 @@
     required: false
     default: None
     aliases: [ 'environment' ]
+    version_added: 2.3
```

Comment:
As it should be handled as string, I would quote the version e.g. `version_added: "2.3"`